vanilla-framework/vanilla-framework
# Documentation content fixes

## Done

- Added 'Basic card' to Card page, and renamed 'Header card' accordingly
- Fixed search box pattern not showing in docs page
- Consolidated form patterns into single 'Forms' page (Forms + Form layout + Form validation)
- Fixed titles for Image page
- Added 'Default link' to links page
- Added missing image position utilities to docs page

## QA

- Pull code
- Run `cd docs && ./run`
- Open http://0.0.0.0:8104/
- Check that basic card has been added to cards page ([issue](https://github.com/vanilla-framework/vanilla-framework/issues/1551))
- Check that search box page loads the example in an iframe
- Check that Forms page contains layouts (inline + stacked) and validation ([issue](https://github.com/vanilla-framework/vanilla-framework/issues/1558))
- Check that Image page displays the correct titles for the examples ([issue](https://github.com/vanilla-framework/vanilla-framework/issues/1573))
- Check that the Links page has an example of a 'Default link' ([issue](https://github.com/vanilla-framework/vanilla-framework/issues/1576))
- Check that the Image Position page has Left and Right included ([issue](https://github.com/vanilla-framework/vanilla-framework/issues/1572))

## Details

Fixes #1551, fixes #1558, fixes #1573, fixes #1576, fixes #1572
